operator/cmd/mesh/manifest-apply.go

```diff
@@ -67,8 +67,8 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
 		RunE: func(cmd *cobra.Command, args []string) error {
 			l := NewLogger(rootArgs.logToStdErr, cmd.OutOrStdout(), cmd.ErrOrStderr())
 			// Warn users if they use `manifest apply` without any config args.
-			if maArgs.inFilename == "" && len(maArgs.set) == 0 && !maArgs.skipConfirmation {
-				if !confirm("This will install the default Istio profile into the cluster. Proceed? (y/N)", cmd.OutOrStdout()) {
+			if !rootArgs.dryRun && !maArgs.skipConfirmation {
+				if !confirm("This will install Istio into the cluster. Proceed? (y/N)", cmd.OutOrStdout()) {
```

Comment:
This should say "..into the cluster, or update Istio if it is already installed."

Reply:
done